### DISCUSSION TITLE
Is it possible to change `max_accepted_htlcs` after opening a channel?

### DISCUSSION THREAD

**Discussion by houtn11** [lightningnetwork/lnd#6999] [Q&A] [answered]
I see there is a `default-remote-max-htlcs` in lnd.config, which is probably used during opening of the channel. 
Is it possible to change `max_accepted_htlcs`  on open channel? It seems `updatechanpolicy` doesn't have that option.

**Comment by yyforyongyu** [comment, marked as answer]
Nope, it's not possible to change that after the channel open.

**Comment by houtn11** [comment]
Default value is 483 htlcs. Does it mean that I might end up, in the worst case force close scenario, with one huge transaction with 483 htlcs on bitcoin?
